common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilter.java

```diff
@@ -81,6 +81,11 @@ int getVersionNumber() {
   public abstract long bitSize();
 
   /**
+   * Swamidass & Baldi (2007) approximation for number of items in a Bloom filter
+   */
+  public abstract double approxItems();
```

Comment:
yea but that would only be off by 1. I wouldn't worry about that since it is approximate anyway.